.github/workflows/test-npgsql.yml

```diff
@@ -43,7 +43,7 @@ jobs:
       matrix:
         os: [ 'ubuntu-latest' ]
         dotnet-version: [ '5.0.x', '6.0.x', '7.0.x' ]
```

Comment:
Seems there is also a new .NET: https://dotnet.microsoft.com/en-us/download/dotnet/8.0

Reply:
Yeah, thanks. Apparently, there is an issue here. The test are actually already running on .NET 8, all of them, that's why they succeeded.

- GH-172

Reply:
I've fixed the test matrix, and added .NET 8.0.

- GH-178

Reply:
When looking at the outcome of the run here [^1], you can immediately spot that more details must be flawed.

![image](https://github.com/crate/cratedb-examples/assets/453543/ed824d6e-7c16-4a79-b74b-299ce5409064)

=> Why does Npgsql succeed on .NET 5? That was not expected to happen!

[^1]: https://github.com/crate/cratedb-examples/actions/runs/7073090667?pr=169

Reply:
I don't have too much energy to "fix" .NET 5 here. I think it will be safe to remove from the test matrix. Please let me know about any objections.

/cc @matriv, @seut, @hlcianfagna, @hammerhead

Reply:
Microsoft ended support for .NET 5 in 2022 so I also think it will be safe to remove it from the test matrix
https://learn.microsoft.com/en-us/lifecycle/products/microsoft-net-and-net-core

Reply:
Thank you for confirming. Support for .NET 5 has been removed with GH-182.

Reply:
Great, let's remove it!